packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -27,6 +27,7 @@ export const debugRenderPhaseSideEffectsForStrictMode = true;
 export const disableInputAttributeSyncing = false;
 export const replayFailedUnitOfWorkWithInvokeGuardedCallback = __DEV__;
 export const warnAboutDeprecatedLifecycles = true;
+export const warnAboutDeprecatedSetNativeProps = false;
```

Comment:
Hm. Is this the canonical one?

Reply:
What do you mean by canonical? It seems like to get Flow to pass these consts have to be defined in every file

Reply:
Ah, nvm. I understand. I need to re-export the one from FeatureFlags. I'm testing this change more closely to ensure I can actually flip this on in FBSource.

Reply:
I think we should actually remove it from the dynamic one if possible. Whenever possible we should always use static flags and drive to remove the dynamic ones.